versions.tf

```diff
@@ -1,3 +1,10 @@
 terraform {
   required_version = ">= 0.12"
```

Comment:
curious why this isn't 0.13 as seen elsewhere

Reply:
Just because it was previously 0.12, and this PR isn't changing anything that would require 0.13. For some of the others, if I didn't know and there wasn't a prior constraint, I just defaulted to 0.13...